openarc/openarc.c

```diff
@@ -1193,7 +1193,7 @@ arcf_config_new(void)
 **  	err -- error string (returned)
 **
 **  Return value:
-**  	TRUE iff the operation succeeded.
+**  	TRUE if the operation succeeded.
```

Comment:
"iff" is right here.  It means "if and only if".

Reply:
Ahh, something learned about the English language :-)